Plugins/Reqnroll.Microsoft.Extensions.DependencyInjection.ReqnrollPlugin/PackageReadme.md

```diff
@@ -0,0 +1,29 @@
+﻿Reqnroll.Microsoft.Extensions.DependencyInjection (pronounced as [reknroʊl]) provides Microsoft.Extensions.DependencyInjection integration for Reqnroll.
```

Comment:
add a link to https://learn.microsoft.com/en-us/dotnet/core/extensions/dependency-injection-usage